src/main/java/uk/gov/companieshouse/insolvency/data/service/InsolvencyServiceImpl.java

```diff
@@ -48,7 +48,7 @@ public void processInsolvency(String contextId, String companyNumber, InternalCo
 
             insolvencyDocumentFromDbOptional
                     .ifPresent(existingDocument -> {
-                                if (!dateFromBodyRequest.isAfter(existingDocument.getDeltaAt())) {
+                                if (dateFromBodyRequest.isBefore(existingDocument.getDeltaAt())) {
```

Comment:
This introduces a bug where if the delta_at in the request is the same as the existing one it will not be considered stale when it should be.

Reply:
I thought that too, but the bug is present regardless. The delta at on the request body is 3 decimal places more accurate than the delta at from the Mongo doc. This means if the delta at is the "same", the request body delta at is always considered more recent.

I presumed fixing this would be out of the scope of the ticket. What do you think?

Reply:
RESOLUTION:
Changing back to `!isAfter`